lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+class Menu 
+  attr_reader :menu_items
+  def initialize
+    @menu_items = {
```

Comment:
This is fine and what was expected from the challenge solution. However, I'd like you to think about something: at the moment, the Menu is hardcoded so all Takeaways will have the same Menu. Can you think of how to make this more flexible such that each Takeaway you initialize can have a different Menu?